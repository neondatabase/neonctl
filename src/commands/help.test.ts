import { describe, expect } from 'vitest';

import { test } from '../test_utils/fixtures';

describe('help', () => {
  test('without args', async ({ testCliCommand }) => {
    await testCliCommand([], {
      stderr: expect.stringContaining(`neonctl <command> [options]`),
    });
  });
});
