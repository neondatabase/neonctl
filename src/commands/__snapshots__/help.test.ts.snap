// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`help > without args 1`] = `""`;
